akka-docs/src/main/paradox/java/howto.md

```diff
@@ -8,8 +8,117 @@ would be nice if every Akka user who finds a recurring pattern in his or her
 code could share it for the profit of all. Where applicable it might also make
 sense to add to the `akka.pattern` package for creating an [OTP-like library](http://www.erlang.org/doc/man_index.html).
 
+@@@ div { .group-java }
+
 You might find some of the patterns described in the Scala chapter of 
-@ref:[HowTo: Common Patterns](howto.md) useful even though the example code is written in Scala.
+@ref:[HowTo: Common Patterns](../scala/howto.md) useful even though the example code is written in Scala.
```

Comment:
that's the page you're on right now, right? so I'd replace `@ref:[HowTo: Common Patterns](howto.md)` with simply "this page".